stream/stream.go

```diff
@@ -80,3 +82,15 @@ type GcpImage struct {
 	Family  string `json:"family,omitempty"`
 	Name    string `json:"name,omitempty"`
 }
+
+// ReplicatedObject represents an object in all regions of an IBMCloud-like cloud
+type ReplicatedObject struct {
+	Regions map[string]RegionObject `json:"regions,omitempty"`
+}
+
+// RegionObject represents an IBMCloud/PowerVS cloud image
+type RegionObject struct {
+	Object string `json:"object,omitempty"`
+	Bucket string `json:"bucket,omitempty"`
+	Url    string `json:"url,omitempty"`
+}
```

Comment:
I just noticed that we need to have a Release field here (note: but not in the release metadata).  Stream metadata records our current recommendation for what image a user should run in different environments, and explicitly supports version skew between platforms.  GCP also omits this due to an oversight; will send a followup PR.

Reply:
GCP fix in https://github.com/coreos/stream-metadata-go/pull/36.

Reply:
ah..ok...done.